index.html

```diff
@@ -90,10 +179,129 @@ <h4>Monday - Friday, February 26th - March 1st, 2024</h4>
+          </div>  <!-- end of col-sm -->
+	  <!-- Following section renders the badge -->
+          <div class="col-sm">
+            <div id="badge">
+              <div id="lname-d">
+                <span id="lname-s"></span>
+              </div>
+              <div id="sname-d">
+                <span id="sname-s"></span>
+              </div>
+              <div id="affili-d">
+                <span id="affili-s"></span>
+              </div>
+              <div id="pronoun-d">
+                <span id="pronoun-s"></span>
+              </div>
+              <div id="event-d">
+                <span id="event-s">Collaboration Meeting, July 21-25, 2025</span>
+              </div>
+              <div id="location-d">
+                <span id="location-s">U. of Illinois/ACP/ICTP-ASIFR</span>
+              </div>
+              <div id="logo1" class="logo"></div>
+            </div>   <!-- closes id="badge" -->
+          </div>  <!-- closes class="col-sm" -->
+        </div>   <!-- closes class="row" -->
+	
+      <div class="container">  <!-- Other in-person stuff -->
+        <h5>COVID Safety</h5>
+	<p>The University of Illinois no longer requires testing or masking for
+	COVID-19, but advises people to follow
+	<a href="https://www.cdc.gov/covid/prevention/index.html" target="_blank"
+	   rel="noopener noreferrer">current CDC guidelines</a>, and to be
+	aware of these
+	<a href="https://www.cdc.gov/respiratory-viruses/prevention/precautions-when-sick.html" target="_blank" rel="noopener noreferrer">
+	  CDC guidelines on the spread of respiratory viruses</a>.</p>
+	  <p>If you feel you are developing symptoms and wish to test, you can
+	  obtain a test kit at
+	  <a href="https://mckinley.illinois.edu/get-healthcare/flucovidresources"
+	     target="_blank" rel="noopener noreferrer">these locations</a>.</p>
+
+	<div class="container" id="UI_info">
+	  <h5>Attendance/Participation</h5>
+	  <div class="form-group">
+	    <div class="form-check">
+	      <input class="form-check-input" type="checkbox" id="de_school" name="de_school">
+	      <label class="form-check-label" for="de_school">
+		<b>Check if you plan to attend DE School in person</b>
+	      </label>
+	    </div>
+	  </div>
+	  <div class="form-group">
+	    <div class="form-check">
+	      <input class="form-check-input" type="checkbox" id="sprint" name="sprint">
+	      <label class="form-check-label" for="de_school">
+		<b>Check if you plan to attend Sprint Day in person</b>
+	      </label>
+	    </div>
+	  </div>
+	  <div class="form-group">
+	    <div class="form-check">
+	      <input class="form-check-input" type="checkbox" id="poster" name="poster">
+	      <label class="form-check-label" for="de_school">
+		<b>Check if you plan to present a poster</b>
+	      </label>
+	    </div>
+	  </div>
+	
+	<h5>Collaboration dinner</h5>
+	<p>The Collaboration dinner will be held on Wednesday, July 23.
+	There is no fee for particiapants; cost is included in registration</p>
```

Comment:
particiapants -> participants

Also, recommend adding "Early career attendees will also be invited to attend a JuDO dinner on Tuesday, July 22."